### ISSUE TITLE
Failed to load the markmap package

### ISSUE THREAD

**Issue by misaka00251** [John-K/inkdrop-markmap#2]
Cannot find module 'prop-types' Require stack:
 - C:\<My Username>\AppData\Roaming\inkdrop\packages\markmap\lib\index.js
 - C:\<My Username>\AppData\Local\inkdrop\app-5.5.3\resources\app.asar\index.html

**Comment by lssch** [comment]
Same on my system

![image](https://user-images.githubusercontent.com/37959269/215519227-e880a271-c10f-4ea8-a41e-a7d40d476ad2.png)

**Comment by smoopmeister** [comment]
Hi folks,
had this myself today and thought i'd share how i fixed it.

Install the module using ipm
`ipm install markmap`

Edit this file
%appdata%\inkdrop\packages\markmap\package.json

Find 
```
"dependencies": {
```

replace the line underneath it, 
```
    "markmap-lib": "0.9.1"
```

with
```
    "markmap-lib": "0.9.1",
    "prop-types": "15.8.1"
```

Run this command to "install modules"
`ipm i`

This will download prop-types and dependencies and you'll have working markmaps

Edit: apologies, forgot to add you need to be in "**%AppData%\Roaming\inkdrop\packages\markmap**" in cm/powershell/terminal etc before you run `ipm i`

**Comment by jayhanjaelee** [comment]
@smoopmeister Thank you, I had same issue.

**Comment by tomashugo** [comment]
> 

@smoopmeister thank you, I had same issue and your fix worked for me too!

**Comment by John-K** [comment]
Thanks folks, sorry for the time taken to get to this. The issue should be fixed by #3
